src/scripts/main.js

```diff
@@ -1,3 +1,43 @@
 'use strict';
 
-// write your code here
+const rows = document.querySelectorAll('tr');
+
+const headerCells = document.querySelectorAll('thead th');
+const secondHeaderCopy = headerCells[1].textContent;
+
+const newHeaderCell = document.createElement('th');
+
+newHeaderCell.textContent = secondHeaderCopy;
+
+const lastHeaderCell = headerCells[headerCells.length - 1];
+
+lastHeaderCell.insertAdjacentElement('beforebegin', newHeaderCell);
+
+rows.forEach((row, index) => {
+  if (index === 0 || index === rows.length - 1) {
```

Comment:
The logic correctly skips the first and last rows, but ensure that there are enough rows to perform the operation. Consider checking if `rows.length` is greater than 2 before proceeding with the loop.